bootstrap_cfn/fab_tasks.py

```diff
@@ -317,3 +319,26 @@ def rsync():
         remote_dir=remote_pillar_dir,
         local_dir=os.path.join(local_pillar_dir, env.environment, '.'),
         use_sudo=True)
+
+@task 
+def update_certs():
```

Comment:
If possible it would be really nice if this method was idempontent - i.e. if the certs haven't changed then running this function should do nothing.

I'm not sure how feasible that is to do thought...